### PR TITLE
Add event listener support

### PR DESCRIPTION
**Pull request by louh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +230,63 @@ var element = geocoder.getContainer();
...
+```
+
+All of Leaflet's event methods are available, such as `on`, `off`, `once`, and so on. The exact syntax and how it behaves are inherited from the version of Leaflet you are plugging into, so there are slight differences between the 0.7.x version line and the 1.0.0 version line.
+
+The following events are fired:
+
+event         | description
+------------- | ---------------------------------------------------------------
+**results**   | Fired when search results are obtained.
+**error**     | Fired if an there was an error with a search request.
```
Comment (dianashk) [inline README.md]: Fired if **_an**_ there was an error with a search request.

**Comment by dianashk** [comment]
Looks great. The event list is short and sweet and they are well documented AND examples are provided! What more could a user ask for!?

File: README.md (hunk 1 above)
Comment (louh) [inline README.md]: :100: thanks!

**Comment by louh** [comment]
Thanks @dianashk ! I'll merge this now.

**Pull request merged**